src/features/student/views/StudentToolsTracesView/components/StudentToolsTracesViewNotice/StudentToolsTracesViewNotice.vue

```diff
@@ -33,11 +37,11 @@ const maxDayBeforeDeletion = computed(() => tracesConfig.value?.maxRemainingDays
 
 <template>
   <div
-    v-if="unassignedTracesSummary && unassignedTracesSummary.total > 0"
+    v-if="unassignedTracesSummary && unassignedTracesSummary?.unassociated > 0"
     class="traces-notice-container"
   >
     <AvNotice
-      :text="createAlertMessage(unassignedTracesSummary.total, unassignedTracesSummary.totalCriticals, criticalDays, maxDayBeforeDeletion)"
+      :text="createAlertMessage(unassignedTracesSummary?.unassociated, unassignedTracesSummary?.totalCriticals, criticalDays, maxDayBeforeDeletion)"
```

Comment:
```suggestion
      :text="createAlertMessage(unassignedTracesSummary.unassociated, unassignedTracesSummary.totalCriticals, criticalDays, maxDayBeforeDeletion)"
```

Grâce au if tu devrais pas avoir besoin des points d'interrogation